tests/transfomers/portable-text-transformer/portable-text-transformer.spec.ts

```diff
@@ -168,6 +278,99 @@ describe("portable text transformer", () => {
     dummyRichText.value = `<p>The coffee drinking culture has been evolving for hundreds and thousands of years. It has never been <a data-item-id="3120ec15-a4a2-47ec-8ccd-c85ac8ac5ba5" href="">so rich as <strong>today</strong></a>. How do you make sense of different types of coffee, what do the particular names in beverage menus mean and what beverage to choose for which occasion in your favorite café?</p>`
     const tree = nodeParse(dummyRichText.value);
     const result = transformToPortableText(tree);
+    expect(result).toMatchSnapshot();
+  })
+
+  it("resolves table cells containing styled fonts", () => {
```

Comment:
Also, I would be in favour of more smaller (unit) tests instead of a one big test that has a table with everything, because it would be easier to see in the snapshots what is what. Since the tests are copy-pasted anyway, you can make a parametrised test. 😉